app/common/directives/mode-bar/mode-bar.html

```diff
@@ -72,6 +72,13 @@
                     <translate translate="nav.login">Log in</translate>
                 </button>
             </li>
+          
+            <li ng-show="currentUser">
+                <a ng-click="viewAccountSettings()">
+                    <img class="avatar" ng-src="https://www.gravatar.com/avatar/{{ currentUser.gravatar || '00000000000000000000000000000000' }}?d=retro" alt="account_settings"/>
+                    <span class="label" translate="nav.account_settings">Account Settings</span>
```

Comment:
Hi @Angamanga I need to clarify something, please help with this. 
- I got a bit confused while trying to resolve the conflict as you asked me to. Line 75-81 is newly added. Want to clarify if I should have totally removed it instead of adding it.

Reply:
- Another thing, I see that lines 71-81 and lines 92-97 are almost exactly the same. The difference between them is only **<span class="label" translate="nav.account_settings">Account Settings</span>** on line 79 and **<span class="label">{{currentUser.realname || currentUser.email}}</span>** on line 95 respectively.

Is line 75-81 meant to stay?

Reply:
Ok, yes there has been changes in the same code since you created this branch. So now we need to fix the code so it works with your changes. If you go to the browser now, you will see two icons with user-avatar when logged in. Line 76-81 ( https://github.com/ushahidi/platform-client/blob/1f0c99e191c6ec74c31f12a013a76b98ebf84633/app/common/directives/mode-bar/mode-bar.html#L76-L81) was made after you created the branch and line 92-97 (https://github.com/ushahidi/platform-client/blob/1f0c99e191c6ec74c31f12a013a76b98ebf84633/app/common/directives/mode-bar/mode-bar.html#L92-L97) was made by you. So what you need to do is adjust line 76-81 so it fits with your changes (changing the <a>-element to a <button> and after that remove line 92-97. You can do that in a new commit and push to this branch. Let me know if you need more guidance or want me to clarify something :)

Reply:
Oh. Ok. Doing this right away. Thanks for clarifying.

Reply:
> So what you need to do is adjust line 76-81 so it fits with your changes (changing the` <a>`-element to a `<button>`) and after that remove line 92-97. You can do that in a new commit and push to this branch. Let me know if you need more guidance or want me to clarify something :)

I have removed 92-97 and changed the a tag to button @Angamanga